Packages/ChatSecure/ChatSecure/Classes/CallKitManager/CallManager.swift

```diff
@@ -155,65 +161,176 @@ final class CallManager: NSObject {
 		NotificationCenter.default.post(name: type(of: self).CallsChangedNotification, object: self, userInfo: userInfo)
 	}
 	
-	func handleIncomingPushEvent(payload: PKPushPayload, completion: ((NSError?) -> Void)? = nil) {
-		//        let jsonData = JSON(payload.dictionaryPayload)
-		//        print("Payload: \(payload.dictionaryPayload)")
-		//        if let currentUserId = jsonData["client_id"].string, let savedCurrentUserId = Multiserver.instance.currentServer.getUserLogin()?.id, currentUserId != savedCurrentUserId {
-		//            print("This call is not belong to me")
-		//            print("\(currentUserId) # \(savedCurrentUserId)")
-		//            return
-		//        }
-		//
-		//        if let username = jsonData["from_client_name"].string,
-		//           let roomId = jsonData["group_id"].string,
-		//           let clientId = jsonData["from_client_id"].string,
-		//           let callType = jsonData["call_type"].string {
-		//            let avatar = jsonData["from_client_avatar"].string
-		//            let token = jsonData["group_rtc_token"].string
-		//            let groupType = jsonData["group_type"].string
-		//            let groupName = jsonData["group_name"].string
-		//
-		//            // Save turnUser and turnPwd
-		//            let turnString = jsonData["turn_server"].stringValue
-		//            let turnData = Data(turnString.utf8)
-		//            do {
-		//                // make sure this JSON is in the format we expect
-		//                if let turnJson = try JSONSerialization.jsonObject(with: turnData, options: []) as? [String: Any] {
-		//                    let turnUser = turnJson["user"] as? String
-		//                    let turnPwd = turnJson["pwd"] as? String
-		//
-		//                    UserDefaults.standard.setValue(turnUser, forKey: Constants.keySaveTurnServerUser)
-		//                    UserDefaults.standard.setValue(turnPwd, forKey: Constants.keySaveTurnServerPWD)
-		//                    UserDefaults.standard.synchronize()
-		//                }
-		//            } catch let error as NSError {
-		//                print("Failed to load: \(error.localizedDescription)")
-		//            }
-		//
-		//            let hasVideo = callType == "video"
-		//            let isGroupCall = groupType == "group"
-		//            let callerName = isGroupCall ? groupName : username
-		//
-		//            reportIncomingCall(isCallGroup: isGroupCall,
-		//                               roomId: roomId,
-		//                               clientId: clientId,
-		//                               avatar: avatar,
-		//                               token: token,
-		//                               callerName: callerName ?? "",
-		//                               hasVideo: hasVideo,
-		//                               completion: completion)
-		//        }
+	public func handleIncomingPushEvent(payload: PKPushPayload, completion: ((NSError?) -> Void)? = nil) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/e5e0a713c2a0f396e41d6374bb8f07759fda2936/Packages/ChatSecure/ChatSecure/Classes/CallKitManager/CallManager.swift#L164"></span>Function body should span 50 lines or less excluding comments and whitespace: currently spans 57 lines<br />
<code>function_body_length</code> <code>CallManager.swift:164</code></td>
    </tr>
  </tbody>
</table>